net/udp/udp_sendto_unbuffered.c

```diff
@@ -198,7 +198,8 @@ static uint16_t sendto_eventhandler(FAR struct net_driver_s *dev,
                      pstate->st_buflen, udpip_hdrsize(pstate->st_conn));
```

Comment:
is it better to return the error from devif_send and related api?

Reply:
@xiaoxiang781216 What do you think of the new `devif_send` prototype?

*(Ignore the build errors, soon I will fix all uses of this function, and properly test this.)*

Reply:
It's enough to change void to int:
```
int devif_send(FAR struct net_driver_s *dev, FAR const void *buf,
                int len, unsigned int offset);
```
maybe all similar function need change.